README.md

```diff
@@ -696,9 +696,9 @@ Layer 4 load balancers look at info at the [transport layer](#communication) to
 
 ### Layer 7 load balancing
 
-Layer 7 load balancers look at the [application layer](#communication) to decide how to distribute requests.  This can involve contents of the header, message, and cookies.  Layer 7 load balancers terminate network traffic, reads the message, makes a load-balancing decision, then opens a connection to the selected server.  For example, a layer 7 load balancer can direct video traffic to servers that host videos while directing more sensitive user billing traffic to security-hardened servers.
+Layer 7 load balancers look at the [application layer](#communication) to decide how to distribute requests.  This can involve contents of the header, message, and cookies.  Layer 7 load balancers terminate network traffic, read the message, make a load-balancing decision, then open a connection to the selected server.  For example, a layer 7 load balancer can direct video traffic to servers that host videos while directing more sensitive user billing traffic to security-hardened servers.
```

Comment:
Because the subject is `Layer 7`, I think these changes are not suitable. In my opinion, we should change this one:
from `Layer 7 load balancers` to `Layer 7 loads balancers`.
Could you consider it again?

Reply:
In this case, that would be incorrect. In the term "load balancer", "load" is a noun, not a verb. It is a component that balances incoming requests (load on the system) across servers. So you wouldn't conjugate "load". The verbs that I changed to the plural third person refer to "balancers", a plural noun.

Reply:
Thank you, I misunderstood, you help me understand it clearly.

Reply:
Welcome 🙂